t/plugin/file-logger3.t

```diff
@@ -0,0 +1,124 @@
+#
```

Comment:
Could we move these tests under https://github.com/apache/apisix/blob/master/t/plugin/file-logger2.t?

Reply:
Move to file-logger2